tests/WE2E/run_WE2E_tests.sh

```diff
@@ -396,6 +396,8 @@ The argument \"machine\" specifying the machine or platform on which to
 run the WE2E tests was not specified in the call to this script.  \
 ${help_msg}"
 fi
+machine=${machine,,}
```

Comment:
Are the changes to this file required?  Throughout the script, `${machine,,}` is used.  These should be replaced with `${machine}` if this update should stay.

Reply:
Yes, I had to fix it here because if one specifies "Hera" instead of "hera" for the machine on the command line, then the workflow module file is not found.  Ok, I will fix the other occurrences.

Reply:
Thanks for addressing the other occurrences of `${machine,,}`!  This looks good to me.